src/main/java/doke/ToDo.java

```diff
@@ -0,0 +1,49 @@
+package doke;
+
+import java.time.LocalDate;
+
+/**
+ * Represent the ToDo task. Extends the Task class.
+ */
+public class ToDo extends Task{
```

Comment:
There should be a space between the Task and {, like so

```suggestion
public class ToDo extends Task {
```